src/Team-Services-Bot.Api/Controllers/MessagesController.cs

```diff
@@ -7,34 +7,56 @@
 // </summary>
 //———————————————————————————————
 
+using System;
 using System.Net;
 using System.Net.Http;
 using System.Threading.Tasks;
 using System.Web.Http;
+using Autofac;
+using Microsoft.ApplicationInsights;
 using Microsoft.Bot.Builder.Dialogs;
 using Microsoft.Bot.Connector;
+using Vsar.TSBot.Dialogs;
 
 namespace Vsar.TSBot.Controllers
 {
     [BotAuthentication]
     public class MessagesController : ApiController
     {
+        private readonly IComponentContext _container;
+        private readonly TelemetryClient _telemetryClient;
+
+        public MessagesController(IComponentContext container, TelemetryClient telemetryClient)
+        {
+            _container = container;
```

Comment:
Are we going to be using the normal StyleCop rules? If so we should not have underscores and should be using "this"